truffle/src/com.oracle.truffle.runtime/src/com/oracle/truffle/runtime/OptimizedCallTarget.java

```diff
@@ -855,6 +858,60 @@ final boolean isCompilationFailed() {
         return compilationFailed;
     }
 
+    final boolean isCompilationStopped() {
+        OptimizedTruffleRuntime.CompilationActivityMode compilationActivityMode = runtime().getCompilationActivityMode();
```

Comment:
Optional: no need to qualify the class name. It makes enough sense without.

Reply:
Bit I'll have to import it then. Nevertheless, I agree that the extra import is worth it in order to simplify this code.